plugin/rpcplugin/hooks_test.go

```diff
@@ -88,7 +126,23 @@ func BenchmarkOnDeactivate(b *testing.B) {
 	}
 }
 
-func BenchmarkOnDeactivate_Unimplemented(b *testing.B) {
+func BenchmarkHooks_ServeHTTP(b *testing.B) {
+	var hooks benchmarkHooks
+
+	if err := testHooksRPC(&hooks, func(remote *RemoteHooks) {
+		b.ResetTimer()
+		for n := 0; n < b.N; n++ {
+			w := httptest.NewRecorder()
+			r, _ := http.NewRequest("POST", "/foo", strings.NewReader("12345678901234567890"))
+			remote.ServeHTTP(w, r)
```

Comment:
~300 microseconds to read the request and write the response. I think we're still on par with typical reverse proxy speeds.